proto/v1/structure/topic_summary.proto

```diff
@@ -0,0 +1,121 @@
+syntax = "proto3";
+
+package org.oppia.proto.v1.structure;
+
+import "org/oppia/proto/v1/structure/image.proto";
+import "org/oppia/proto/v1/versions/structure_versions.proto";
+
+option java_package = "org.oppia.proto.v1.structure";
+option java_multiple_files = true;
+
+// Represents the summary of a topic that is available to the user to download. A topic is a fully
+// contained module of lessons, explanations, and practice questions to learn a specific subcategory
+// of a subject area (such as 'place values' or 'fractions').
+//
+// Topics organize lessons into stories which organize teaching around a plot to help contextualize
+// the lessons and to make them more engaging to learners.
+//
+// Topics organize 'skills' (i.e. granular units of learning) into groups called 'subtopics' which
+// have corresponding revision cards that learners may use to get a high-level refresher of the
+// corresponding skills.
+message DownloadableTopicSummary {
```

Comment:
@BenHenning Could you please provide some reference on below few points:
1. The proto size of the `DownloadableTopicSummary` is the size which we show in the app as the downloadable size, right?
2. From the web codebase, we have files - classes -> 
     - `topic_domain.py` - `class TopicSummary`
     - `story_domain.py` - `class StorySummary`
     - `skill_domain.py` - `class SkillSummary`
     - `subtopic_page_domain.py` - `class SubtopicPage`
     - How should we create one single proto for `DownloadableTopicSummary` from these above-mentioned different classes?
 
According to this, we need to fix the below-mentioned sub-issues in one single issue:
1. https://github.com/oppia/oppia/issues/13828
2. https://github.com/oppia/oppia/issues/13827
3. https://github.com/oppia/oppia/issues/13826
4. https://github.com/oppia/oppia/issues/13349